osmose_config.py

```diff
@@ -284,11 +284,9 @@ class gen(default_country):
     # Bordeaux
     'merge_recycling_FR_bm',
     'merge_parking_FR_bm',
-    'merge_bicycle_parking_FR_bordeaux',
```

Comment:
Even if the OpenData page is not available anymore, the source file [still is](http://opendatabdx.cloudapp.net/DataBrowser/DownloadCsv?container=databordeaux&entitySet=sigstavelo&filter=NOFILTER). Wouldn't it be better to keep it until the source file is removed ?

Reply:
You are right. The are data on Bordeaux Metropole.
https://opendata.bordeaux-metropole.fr/explore/dataset/st_arceau_p/information/
But I misinterpret the role of merge_parking_FR_bm with is only for car.
We need to add a new analyser for Bordeaux Metropole in place.

Reply:
#1170